recipe/meta.yaml

```diff
@@ -0,0 +1,151 @@
+{% set name = "torchaudio" %}
+{% set version = "2.1.2" %}
+{% set run_unittests = False %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pytorch/audio/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 82c2745a73172b495327ec36c6970ad5ad9d5d2ac44feeaea2617152f9393bf7
+
+build:
+  number: 0
+  skip: True  # [py>311]
+  script:
+    - export USE_FFMPEG=ON                         # [not win]
+    - set USE_FFMPEG=ON                            # [win]
+    - export BUILD_SOX=OFF                         # [not win]
+    - set BUILD_SOX=OFF                            # [win]
+    - export USE_OPENMP=ON                         # [not win]
+    - set USE_OPENMP=ON                            # [win]
+    - export BUILD_TORCHAUDIO_PYTHON_EXTENSION=ON  # [not win]
+    - set BUILD_TORCHAUDIO_PYTHON_EXTENSION=ON     # [win]
+    - export FFMPEG_ROOT=${PREFIX}                 # [not win]
+    - set FFMPEG_ROOT=%LIBRARY_PREFIX%             # [win]
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  missing_dso_whitelist:
+    - "**/libc10.so"                # [linux]
+    - "**/libtorch_cpu.so"          # [linux]
+    - "**/libtorch_python.so"       # [linux]
+    - "**/libtorchaudio_ffmpeg.so"  # [linux or osx]
+    - "**/libtorchaudio.so"         # [linux or osx]
+    - "**/ld64.so.1"                # [s390x]
+    - "**/libc10.dylib"             # [osx]
+    - "**/libtorch_cpu.dylib"       # [osx]
+    - "**/libtorch_python.dylib"    # [osx]
+    - "**/torch_cpu.dll"            # [win]
+    - "**/c10.dll"                  # [win]
+    # because of pytorch --> _openmp_mutex --> libgomp
+    - "**/lib/libgomp.so.*"         # [linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - make  # [not win]
+    - patchelf  # [linux]
+    - llvm-openmp  # [osx]
+    - intel-openmp  # [not osx and blas_impl == 'mkl']
```

Comment:
These should be moved into `host` as we need to link to them.

Reply:
done!